### PR TITLE
added golang helloworld

### PR DESCRIPTION
**Pull request by modrake**
This code is important, otherwise world might feel lonely.

### CODE REVIEW

File: mo-helloworld/hello.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Simple program for teaching git flow and code review
+package main
+
+import "fmt"
+
+func main() {
```
Comment (beandrake) [inline mo-helloworld/hello.go:6]: Why on earth did you name this function main?

**Review by beandrake — CHANGES_REQUESTED** [review]

File: mo-helloworld/hello.go (hunk 1 above)
Comment (modrake) [reply mo-helloworld/hello.go:6]: because if I didn't I couldn't run the code :shipit: ?

Comment (modrake) [reply mo-helloworld/hello.go:6]: we should implement a python version tho

Comment (beandrake) [reply mo-helloworld/hello.go:6]: You reasoning is sound, but ultimately I agree with you that a Python version would be better.

**Comment by beandrake** [comment]
Pull request denied, for funsies.

**Pull request closed without merging**